src/tools/manifest2proto.ts

```diff
@@ -69,7 +81,7 @@ async function particleSpecToProtoPayload(spec: ParticleSpec) {
 }
 
 async function recipeToProtoPayload(recipe: Recipe) {
-  assert(recipe.normalize());
+  recipe.normalize();
```

Comment:
I'm curious, why remove the assertion here? Does this code correctly handle un-normalised recipes?

Reply:
normalize() returns whether it performed normalization. If a recipe was already normalized then it returns false, which was making this function fail.

Reply:
I didn't realise this, thanks for explaining.